generated/java/gapic-google-cloud-dlp-v2beta1/src/main/java/com/google/cloud/dlp/v2beta1/DlpServiceClient.java

```diff
@@ -0,0 +1,770 @@
+ *
+ * <p>The surface of this class includes several types of Java methods for each of the API's
+ * methods:
+ *
+ * <ol>
+ *   <li> A "flattened" method. With this type of method, the fields of the request type have been
+ *       converted into function parameters. It may be the case that not all fields are available as
+ *       parameters, and not every API method will have a flattened method entry point.
+ *   <li> A "request object" method. This type of method only takes one parameter, a request object,
+ *       which must be constructed before the call. Not every API method will have a request object
+ *       method.
+ *   <li> A "callable" method. This type of method takes no parameters and returns an immutable API
+ *       callable object, which can be used to initiate calls to the service.
+ * </ol>
+ *
+ * <p>See the individual methods for example code.
+ *
+ * <p>Many parameters require resource names to be formatted in a particular way. To assist with
+ * these names, this class includes a format method for each type of name, and additionally a parse
+ * method to extract the individual identifiers contained within names that are returned.
+ *
+ * <p>This class can be customized by passing in a custom instance of DlpServiceSettings to
+ * create(). For example:
+ *
+ * <pre>
+ * <code>
+ * DlpServiceSettings dlpServiceSettings =
+ *     DlpServiceSettings.defaultBuilder()
+ *         .setCredentialsProvider(FixedCredentialsProvider.create(myCredentials))
+ *         .build();
+ * DlpServiceClient dlpServiceClient =
+ *     DlpServiceClient.create(dlpServiceSettings);
+ * </code>
+ * </pre>
+ */
+@Generated("by GAPIC")
+@BetaApi
+public class DlpServiceClient implements AutoCloseable {
+  private final DlpServiceSettings settings;
+  private final ScheduledExecutorService executor;
+  private final ManagedChannel channel;
+  private final OperationsClient operationsClient;
+  private final List<AutoCloseable> closeables = new ArrayList<>();
+
+  private final UnaryCallable<InspectContentRequest, InspectContentResponse> inspectContentCallable;
+  private final UnaryCallable<RedactContentRequest, RedactContentResponse> redactContentCallable;
+  private final UnaryCallable<CreateInspectOperationRequest, Operation>
+      createInspectOperationCallable;
+  private final OperationCallable<
+          CreateInspectOperationRequest, InspectOperationResult, InspectOperationMetadata>
+      createInspectOperationOperationCallable;
+  private final UnaryCallable<ListInspectFindingsRequest, ListInspectFindingsResponse>
+      listInspectFindingsCallable;
+  private final UnaryCallable<ListInfoTypesRequest, ListInfoTypesResponse> listInfoTypesCallable;
+  private final UnaryCallable<ListRootCategoriesRequest, ListRootCategoriesResponse>
+      listRootCategoriesCallable;
+
+  private static final PathTemplate RESULT_PATH_TEMPLATE =
+      PathTemplate.createWithoutUrlEncoding("inspect/results/{result}");
```

Comment:
pushed a new commit based on [this](https://github.com/googleapis/googleapis/pull/390). PTAL